checks/network.py

```diff
@@ -73,3 +73,10 @@ def checkDynamicBitrate(lines):
             return [LEVEL_WARNING, "Dynamic Bitrate",
                     """Dynamic Bitrate is enabled and a hardware encoder is potentially in use. This can cause issues with hardware encoders if bitrate changes happen too frequently, or drops too low. Should you experience bitrate dropping to zero, or no output even if OBS says its streaming, either change your Encoder to x264 or turn off Dynamic Bitrate in Settings -> Advanced -> Network."""]
     return None
+
+
+def checkStreamDelay(lines):
+    delayLines = search('second delay active', lines)
+    if (len(delayLines) > 0):
+        return [LEVEL_INFO, "Stream Delay", "Stream Delay may currently be active. This means that your stream is being delayed by a certain number of seconds. If this is not what you intended, please go to Settings -> Advanced -> Stream Delay, and uncheck the Enable box"]
```

Comment:
I think people know how to disable a checkbox, so just "this can be disabled in Settings -> etc" is fine.

Additionally, please be sure to end the sentence with a dot.

Reply:
One of my intentions for this is the fact that it seems like quite a few people unintentionally enable it, and I'm worried that those exact people I'm targeting for this might not know how to disable/enable it.